luma/lcd/device.py

```diff
@@ -649,6 +649,168 @@ def command(self, cmd, *args):
             self._serial_interface.data(list(args))
 
 
+class ili9486(backlit_device, __framebuffer_mixin):
+    """
+    Serial interface to a 262k color (6-6-6 RGB) ILI9486 LCD display.
+
+    On creation, an initialization sequence is pumped to the display to properly
+    configure it. Further control commands can then be called to affect the
+    brightness (if implemented) and other settings.
+
+    Note that the ILI9486 display used for development -- a Waveshare
+    3.5-inch IPS LCD(B) -- used a portrait orientation.  Images were
+    rendered correctly only when specifying that height was 480 pixels
+    and the width was 320.
+
+    :param serial_interface: the serial interface (usually a
+        :py:class:`luma.core.interface.serial.spi` instance) to delegate sending
+        data and commands through.
+    :param width: The number of pixels laid out horizontally.
+    :type width: int
+    :param height: The number of pixels laid out vertically.
+    :type height: int
+    :param rotate: An integer value of 0 (default), 1, 2 or 3 only, where 0 is
+        no rotation, 1 is rotate 90° clockwise, 2 is 180° rotation and 3
+        represents 270° rotation.
+    :type rotate: int
+    :param framebuffer: Framebuffering strategy, currently instances of
+        ``diff_to_previous()`` or ``full_frame()`` are only supported.
+    :type framebuffer: luma.core.framebuffer.framebuffer
+    :param bgr: Set to ``True`` if device pixels are BGR order (rather than RGB).
+    :type bgr: bool
+    :param h_offset: Horizontal offset (in pixels) of screen to device memory
+        (default: 0).
+    :type h_offset: int
+    :param v_offset: Vertical offset (in pixels) of screen to device memory
+        (default: 0).
+    :type v_offset: int
+
+    .. versionadded:: ???
```

Comment:
I’ll fill in the version no. when I prepare the release

Reply:
Ok, do I need to take any other action?